inspirehep/bat/pages/create_literature.py

```diff
@@ -454,6 +454,40 @@ def _submit_doi_id():
     return ArsenicResponse(_submit_doi_id)
 
 
+def import_and_submit_arxiv_by_id(arxiv_id):
+    def _import_and_submit_arxiv_by_id():
+        return (
+            'The INSPIRE staff will review it and your changes will be added '
+            'to INSPIRE.'
+        ) in WebDriverWait(Arsenic(), 10).until(
+            EC.visibility_of_element_located(
+                (
+                    By.XPATH,
+                    '(//div[@class="alert alert-success alert-form-success"])',
+                )
+            )
+        ).text
+    Arsenic().find_element_by_id('arxiv_id').send_keys(arxiv_id)
+    WebDriverWait(Arsenic(), 10).until(
+        EC.visibility_of_element_located((By.ID, 'importData'))
+    ).click()
+    WebDriverWait(Arsenic(), 20).until(
+        EC.visibility_of_element_located((By.ID, 'acceptData'))
+    ).click()
+    WebDriverWait(Arsenic(), 20).until(
+        EC.visibility_of_element_located((By.ID, 'arxiv_id'))
+    )
+
+    _skip_import_data()
+
+    Arsenic().find_element_by_xpath(
+        '//div[@id="webdeposit_form_accordion"]/div[4]/span/button'
+    ).click()
+    Arsenic().show_title_bar()
+
+    return ArsenicResponse(_import_and_submit_arxiv_by_id)
```

Comment:
Why is a PR that add supports for files doing this?

Reply:
I know but this was a refactoring needed to support additional acceptance tests by reusing components. Everything was hardcoded before.